src/main/java/uk/gov/companieshouse/api/accounts/request/AccountsNoteConverter.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse.api.accounts.request;
```

Comment:
Test class ?

Reply:
Added test class however it uses EnumMap so i could not find a way to hit the if clause where it would throw exception and the code coverage is not up to what we would expect it to be